rj_gameplay/stp/skill/action_behavior.py

```diff
@@ -3,28 +3,38 @@
 import stp.action as action
 from typing import Optional, Any
 import stp.rc as rc
+from typing import TypedDict, List
 
+class RobotActions(TypedDict):
```

Comment:
Out of curiosity, does this change break other gameplay actions that are already merged in? (I guess that's something you can't answer until you try merging in ros2.)

Reply:
No it shouldn't

Reply:
Great